src/utils.js

```diff
@@ -73,3 +73,23 @@ export function listAllFields(rules) {
   });
   return allFields.flat();
 }
+
+export function removeFieldValue(path, formData = {}) {
+  let separator = path.indexOf(".");
+
+  if (separator === -1) {
+    formData[path] = undefined;
+    // set(formData, path, undefined);
```

Comment:
Can this be removed or should it be used?

Reply:
it can be removed